### DISCUSSION TITLE
vscode: Removing the quick fix entry for adding to dictionary

### DISCUSSION THREAD

**Discussion by losnappas** [streetsidesoftware/vscode-spell-checker#2421] [Q&A] [answered]
I couldn't find a setting for not showing those 2 options. Does a setting like that exist?

E.g. the last 2 here, I'd rather not show those because often they get in the way:

![image](https://user-images.githubusercontent.com/25119292/210135301-5f12d3fa-7822-471d-bdd0-bb9f849821a6.png)

**Comment by Jason3S** [comment, marked as answer]
@losnappas,

There isn't currently an option to hide those. Please feel free to make an enhancement request.

**Reply by losnappas** [reply]
https://github.com/streetsidesoftware/vscode-spell-checker/issues/2430
